products/qt.md

```diff
@@ -16,6 +16,13 @@ auto:
 -   git: https://github.com/qt/qt5.git
 
 releases:
+-   releaseCycle: "6.6"
+    releaseDate: 2023-10-09
```

Comment:
```suggestion
    releaseDate: 2023-10-10
```

According to https://www.qt.io/blog/qt-6.6-released. Will be overwritten by our automation but at least it will be in the git history.